plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md

```diff
@@ -478,3 +493,61 @@ spec:
 ```
 
 // output: `"\"hello\": \"world\""`
+
+### Merge Json
+
+Action: `roadiehq:utils:json:merge`
+
+**Required params:**
+
+- path: The file path for the json you want to edit.
+- content: The Json you want to merge in. Can be either an object or a string.
+
+```yaml
+---
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: merge-json-into-file
```

Comment:
I dont see how this is merging? It seems to be writing a string content to a file?

Reply:
Ah sorry getting changes mixed up.